UnitystationLauncher/ViewModels/ChangelogViewModel.cs

```diff
@@ -1,41 +1,52 @@
+using System.Collections.Generic;
 using ReactiveUI;
 using Octokit;
 using System.Collections.ObjectModel;
 using System.Reactive.Concurrency;
+using System.Threading;
 using System.Threading.Tasks;
+using Serilog;
 
 namespace UnitystationLauncher.ViewModels
 {
     public class ChangelogViewModel : ViewModelBase
     {
-        readonly GitHubClient _client;
-        ObservableCollection<PullRequestViewModel> _pullRequests = new ObservableCollection<PullRequestViewModel>();
+        private readonly GitHubClient _client;
 
-        public ObservableCollection<PullRequestViewModel> PullRequests
-        {
-            get => _pullRequests;
-            set => this.RaiseAndSetIfChanged(ref _pullRequests, value);
-        }
+        ObservableCollection<PullRequestViewModel> PullRequests { get; }
 
         public ChangelogViewModel()
         {
             _client = new GitHubClient(new ProductHeaderValue("UnitystationCommitNews"));
-            RxApp.MainThreadScheduler.ScheduleAsync((scheduler, ct) => GetPullRequestsAsync());
+            PullRequests = new();
+
+            RxApp.TaskpoolScheduler.ScheduleAsync(GetPullRequestsAsync);
         }
 
-        public async Task GetPullRequestsAsync()
+        private async Task GetPullRequestsAsync(IScheduler scheduler, CancellationToken cancellationToken)
```

Comment:
```suggestion
        private async Task GetPullRequestsAsync(IScheduler _, CancellationToken cancellationToken)
```

this should suffice to comply with the signature and communicate it is an unused parameter

Reply:
Ah yeah thanks, should be fixed now.